src/main/java/de/tum/in/www1/artemis/web/rest/dto/CourseWithIdDTO.java

```diff
@@ -0,0 +1,4 @@
+package de.tum.in.www1.artemis.web.rest.dto;
+
+public record CourseWithIdDTO(long id) {
```

Comment:
Question not related to this PR but worth a discussion:
@DominikRemo do you know if there is a way to disable this SCA rule for DTOs? This will come up regularly in the future.

Reply:
At least for files in the dto package this should be fairly easy right?

Reply:
This is a codacy rule but probably it's similar to teamscale.